tests/misc_testsuite/export-large-signature.wast

```diff
@@ -0,0 +1,5 @@
+(module
+  (func (export "many_params")
+    (param f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32 f32)
+  )
+)
```

Comment:
@alexcrichton, I shouldn't need to assert anything here, right? Wasmtime will still attempt to compile this inside [`module(...)`](https://github.com/bytecodealliance/wasmtime/pull/1318)?

Reply:
I believe so yeah, but it might be good to double-check by running the tests before this PR as well to make sure this test fails.